woocommerce/woocommerce-gutenberg-products-block
# Fix: remove IntersectionObserver shim

As WordPress 5.8 dropped support for IE11, so does WooCommerce, this PR removes the shim for `IntersectionObserver`. This PR also converts the source file to typescript.

Fixes #3035 




#### Accessibility




### Screenshots



### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. Check out this PR.
2. Run `npm run dev`.
3. Test the Cart block, see no error in console.



### Changelog

> Remove IntersectionObserver shim in favor of dropping IE11 support.
